config/types_azure.go

```diff
@@ -174,9 +174,9 @@ type AzureFrontdoorSettings struct {
 	DnsResourceGroup string                   `yaml:"dns_resource_group"`
 	SslKeyVault      *AzureFrontdoorSslConfig `yaml:"ssl_key_vault"`
 
-	// Undocumented option to workaround some tenacious issues
+	// Undocumented option to work around some tenacious issues
 	// with using Frontdoor in the Azure Terraform provider
-	SupressChanges bool `yaml:"supress_changes"`
+	SuppressChanges bool `yaml:"suppress_changes"`
```

Comment:
This does introduces a backwards incompatible change: https://github.com/labd/mach-composer/blob/2ec4c78c4943b37aab684b0d56a7292f7444e1da/config/types_azure.go#L179

But i'm fine with it

Reply:
Actually the documentation already says it should be what I changed it into; https://docs.machcomposer.io/reference/syntax/global.html#frontdoor

Reply:
Indeed, was also implemented like that 
<img width="480" alt="afbeelding" src="https://user-images.githubusercontent.com/1533933/169522655-7cc3e5bf-b6ce-431e-b8d2-48ee60cd739a.png">